include/libp2p/multi/multiaddress_protocol_list.hpp

```diff
@@ -120,15 +123,15 @@ namespace libp2p::multi {
         {Protocol::Code::IP6, 128, "ip6"},
         {Protocol::Code::IP6_ZONE, Protocol::kVarLen, "ip6zone"},
         {Protocol::Code::DNS, Protocol::kVarLen, "dns"},
-        {Protocol::Code::DNS4, Protocol::kVarLen, "dns64"},
+        {Protocol::Code::DNS4, Protocol::kVarLen, "dns4"},
         {Protocol::Code::DNS6, Protocol::kVarLen, "dns6"},
         {Protocol::Code::DNS_ADDR, Protocol::kVarLen, "dnsaddr"},
         {Protocol::Code::SCTP, 16, "sctp"},
         {Protocol::Code::UDT, 0, "udt"},
         {Protocol::Code::UTP, 0, "utp"},
         {Protocol::Code::UNIX, Protocol::kVarLen, "unix"},
-        // {Protocol::Code::P2P, Protocol::kVarLen, "p2p"},
-        {Protocol::Code::P2P, Protocol::kVarLen, "ipfs"},
+        {Protocol::Code::P2P, Protocol::kVarLen, "p2p"},
+//        {Protocol::Code::P2P, Protocol::kVarLen, "ipfs"},
```

Comment:
why not remove this completely ?

Reply:
In my view, this either has to be removed completely (due to multiaddress_protocol_list.hpp:81) or left uncommented (due to converter_utils.cpp:133)